src/assets/scss/component/_custom-form-styles.scss

```diff
@@ -58,502 +55,494 @@ $formContainerBorderColor: #ccc;
-  padding: 7px 7px 7px 27px;
-  text-decoration: none;
-}
-
-.disclaimer .ss-terms a:link {
-  color: $disclaimerLinkColor;
-  text-decoration: none;
-}
-.disclaimer .ss-terms a:hover {
-  text-decoration: underline
-}
-.disclaimer .ss-terms a:active {
-  color: $disclaimerLinkActiveColor
-}
-.disclaimer .ss-terms a:visited {
-  color: $disclaimerLinkVisitedColor
-}
-
-.disclaimer {
-  color: $disclaimerColor;
-  font: normal 11px arial,sans-serif;
-  margin-top: 11px;
-}
-.disclaimer-separator {
-  border-top: 1px solid $disclaimerSeparatorBorderColor
-}
-
-.disclaimer .ss-logo-link {
-  color: $disclaimerLogoLinkColor;
-  text-decoration: none;
-}
-.powered-by-logo {
-  display: inline-block;
-  vertical-align: bottom;
-}
-.powered-by-text {
-  line-height: 16px
-}
-.ss-terms {
-  display: inline-block;
-  line-height: 20px;
-  width: 80%;
-  text-align: center;
-}
-.ss-submit-message {
-  font: normal 13px arial,sans-serif;
-  margin-bottom: 20px;
-}
-
-.ss-form-container {
-  background-color: $formContainerColor;
-  border: 1px solid $formContainerBorderColor;
-  margin: 0.5em auto 0;
-  padding: 1em;
-  width: 45em;
-}
-
-.ss-editing-response {
-  border: 1px solid $editingResponseBorderColor;
-  margin: 1em auto;
-  padding: 0 1em;
-  width: 43em;
-}
-
-.ss-confirmation-banner, .ss-editing-response {
-  background-color: $confirmationBannerBackground
-}
-.errorbox-bad {
-  border: 2px solid $error;
-  background-color: $errorBoxBadBorderColor;
-  padding: 2px;
-}
-.error-message {
-  display: none;
-}
-
-.errorheader {
-  color: $error
-}
-input#emailReceipt {
-  margin-left: 0;
-  margin-top: 0;
-  vertical-align: middle;
-}
-.no-chrome .powered-by-logo, .no-chrome .ss-legal, .no-chrome .ss-edit-link {
-  display: none
-}
-.ss-prepopulate-input, .ss-submit-url-input {
-  width: 100%
+.google-form {
+  html .goog-inline-block,
+  *:first-child+html .goog-inline-block {
+    display: inline;
+  }
+  body {
+    color: $baseColor;
+    font-family: $bodyFontFamily;
+    font-weight: 400;
+    margin: 0;
+    line-height: $bodyLineHeight;
+    padding: 0 1em;
+    background-color: white;
+    font-weight: normal;
+  }
+  input,
+  textarea {
+    outline: none;
+  }
+  .container {
+    margin: 0 auto;
+    width: 960px;
+  }
+  .ss-logo-link {
+    display: block;
+  }
+  .ss-logo {
+    border: none;
+    height: 23px;
+    width: 105px;
+  }
+  .g {
+    color: $baseColor;
+  }
+  .ss-form-desc {
+    font: inherit;
+    width: 99%;
+    margin: 0;
+  }
+  .ss-powered-by {
+    display: block;
+    clear: left;
+    color: $baseColor;
+    margin: 1em 0.2em 0.2em;
+  }
+  .ss-powered-by a:link,
+  .ss-powered-by a:visited {
+    color: $baseColor;
+  }
+  .ss-required-asterisk,
+  .ss-required {
+    color: $error;
+  }
+  .ss-no-ignore-whitespace {
+    white-space: pre-wrap;
+    word-wrap: break-word;
+  }
+  .ss-confirmation-header {
+    font-size: 140%;
+    padding-bottom: 1em;
+  }
+  .ss-abuse-header {
+    background: #fff68f;
```

Comment:
Color literals like `#fff68f` should only be used in variable declarations; they should be referred to via variable everywhere else.